api/arceos_posix_api/src/imp/time.rs

```diff
@@ -56,6 +56,23 @@ pub unsafe fn sys_clock_gettime(_clk: ctypes::clockid_t, ts: *mut ctypes::timesp
     })
 }
 
+/// Get clock time since booting
+pub unsafe fn sys_clock_settime(_clk: ctypes::clockid_t, ts: *mut ctypes::timespec) -> c_int {
+    syscall_body!(sys_clock_setttime, {
+        if ts.is_null() {
+            return Err(LinuxError::EFAULT);
+        }
+        let new_tv = Duration::from(*ts);
+        axhal::time::set_current_time(new_tv);
+        debug!(
```

Comment:
`debug` information should always appear before specific operations, in order to checkout whether input messages are wrong or not. Here `debug` should be put before `axhal::time::set_current_time(new_tv)`.